demos/http/http_demo_mutual_tls/demo_config.h

```diff
@@ -0,0 +1,137 @@
+ * subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in all
+ * copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS
+ * FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR
+ * COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER
+ * IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN
+ * CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+ */
+
+#ifndef DEMO_CONFIG_H_
+#define DEMO_CONFIG_H_
+
+/**************************************************/
+/******* DO NOT CHANGE the following order ********/
+/**************************************************/
+
+/* Logging related header files are required to be included in the following order:
+ * 1. Include the header file "logging_levels.h".
+ * 2. Define LIBRARY_LOG_NAME and  LIBRARY_LOG_LEVEL.
+ * 3. Include the header file "logging_stack.h".
+ */
+
+/* Include header that defines log levels. */
+#include "logging_levels.h"
+
+/* Logging configuration for the Demo. */
+#define LIBRARY_LOG_NAME     "DEMO"
+#define LIBRARY_LOG_LEVEL    LOG_INFO
+
+#include "logging_stack.h"
+
+/************ End of logging configuration ****************/
+
+/**
+ * @brief HTTP server host name.
+ */
+#define SERVER_HOST                "server.cryptomix.com"
+
+/**
+ * @brief The length of the HTTP server host name.
+ */
+#define SERVER_HOST_LENGTH         ( sizeof( SERVER_HOST ) - 1 )
+
+/**
+ * @brief HTTP server port number.
+ *
+ * In general, port 443 is for TLS HTTP connections.
+ */
+#define SERVER_PORT                443
+
+/**
+ * @brief Path of the file containing the server's root CA certificate for TLS authentication.
+ *
+ * @note This certificate should be PEM-encoded.
+ */
+#define SERVER_CERT_PATH           "certificates/dst.crt"
+
+/**
+ * @brief Length of path to server certificate.
+ */
+#define SERVER_CERT_PATH_LENGTH    ( ( uint16_t ) ( sizeof( SERVER_CERT_PATH ) - 1 ) )
+
+/**
+ * @brief Path of the file containing the client's certificate for TLS authentication.
+ *
+ * For the purposes of this demo, a self-signed certificate is used. However,
+ * in production, this certificate should be signed by a certifying authority
+ * that is trusted by the server.
+ *
+ * @note This certificate should be PEM-encoded.
+ */
+#define CLIENT_CERT_PATH           "certificates/client.crt"
+
+/**
+ * @brief Length of path to client's certificate.
+ */
+#define CLIENT_CERT_PATH_LENGTH    ( ( uint16_t ) sizeof( CLIENT_CERT_PATH ) - 1 )
+
+/**
+ * @brief Path of the file containing the client's key for TLS client authentication.
+ *
+ *
+ * @note This key should be PEM-encoded.
+ */
+#define CLIENT_KEY_PATH            "certificates/client.key"
```

Comment:
Assuming the same changes as the other PR are going to made for configurable macros here too.